packages/tools/src/tools/CrosshairsTool.ts

```diff
@@ -1108,7 +1108,14 @@ class CrosshairsTool extends AnnotationTool {
 
       let color =
         viewportColor !== undefined ? viewportColor : 'rgb(200, 200, 200)';
+
       let lineWidth = 1;
+
+      if (/Mobi|Android/i.test(navigator.userAgent)) {
```

Comment:
can we put it in the constructor and assign it to a private property?

Reply:
Hey @sedghi great idea. I've changed it. Could you check it? Is there anything else to change? Thanks.